ra/registration-authority_test.go

```diff
@@ -401,12 +401,14 @@ func TestNewRegistrationNoFieldOverwrite(t *testing.T) {
 	//test.Assert(t, result.Agreement != "I agreed", "Agreement shouldn't be set with invalid URL")
 
 	id := result.ID
```

Comment:
Let's pull out this `id` alias. I think it winds up adding confusion, and the code below would be clearer with `AssertNotEquals(t, result.ID, badReg.ID)`.